lib/vhdl-linter.ts

```diff
@@ -57,7 +57,7 @@ export class VhdlLinter {
             return actions;
           });
         }
-
+        this.messages.push(...this.parser.state.messages);
```

Comment:
where does this come from? The parser messages are added in the parser rule

Reply:
Added comment